src/components/Search/SearchList.tsx

```diff
@@ -295,8 +304,77 @@ function SearchList(
 
     useImperativeHandle(ref, () => ({scrollAndHighlightItem, scrollToIndex}), [scrollAndHighlightItem, scrollToIndex]);
 
+    const getItemHeight = useCallback(
```

Comment:
So this means that if we add a new list item, we'd have to add it here too right? Is there a more foolproof way of doing this? Maybe we could use the same pattern we have for sections, e.g. [here](https://github.com/Expensify/App/blob/e276b25f4c358313ecc076121e2afeb802ce6bbb/src/libs/SearchUIUtils.ts#L655-L670), so that it's centralized in SearchUIUtils and this component just calls `getItemHeight(type, ...)`. I think that'd make it easier to see all the places that we need to update when adding a new list item type

Reply:
I agree that we can architect something neater. I have a few concerns about putting it away since this value will always recalculate on resizing or when new data comes in. This may cause some loading effects.
The second thing is that our items sometimes have three different sizes (wide screen with drawer, small screen with drawer, small screen) and sometimes it's messy and undesired. If we can unify some sizes of items, we can drastically simplify it

Reply:
Yea, let's clean this up in a follow up. I think that we can make it easier to scale as we add more items to Reports